doc/nrf/protocols/matter/end_product/security.rst

```diff
@@ -7,14 +7,44 @@ Security
    :local:
    :depth: 3
 
-Nordic Matter samples leverage security features supported in the |NCS| that can be divided into three major categories:
+Nordic Matter samples leverage :ref:`security` features supported in the |NCS| that can be divided into four major categories:
 
+* Secure processing environment
 * Cryptography
 * Trusted storage
 * Securing production devices
 
 In the following sections you will learn more details about each listed category.
 
+Secure processing environment
+*****************************
+
+Depending on the board, Matter samples can use a secure processing environment.
+
+nRF54L with Trusted Firmware-M (TF-M)
+=====================================
+
+On the nRF54L SoC, Matter samples support :ref:`app_boards_spe_nspe` with Trusted Firmware-M (TF-M).
+All cryptographic operations within the Matter stack are performed by utilizing the `Platform Security Architecture (PSA)`_ API and executed in the secure TF-M environment.
+The secure materials like Matter Session keys, DAC private key and other keys, are stored in the TF-M secure storage utilizing the :ref:`tfm_encrypted_its` module.
+Matter samples utilize the full TF-M library, so you cannot use the :ref:`tfm_minimal_build` version of TF-M.
+
+To build a Matter sample with the TF-M support add the ``/ns`` variant to the board target.
+For example, use the following command for the nRF54L15 SoC:
+
+.. code-block:: console
+
+   west build -b nrf54l15dk/nrf54l15/cpuapp/ns
+
+To configure partition layout, configure the :file:`pm_static_nrf54l15dk_nrf54l15_cpuapp_ns.yml` file that is available in each sample directory.
```

Comment:
This file is used automatically and there is no need to invoke it manually in the build command, correct?

Reply:
Yes, but if users want to change the partition layout this is a hint for them to edit this file.

Reply:
```suggestion
To configure partition layout for your application, you can edit the :file:`pm_static_nrf54l15dk_nrf54l15_cpuapp_ns.yml` file that is available in each sample directory.
```